### PR TITLE
Pre-release updates

### PR DESCRIPTION
**Pull request by trallard**
This PR updates the changelog in preparation for the upcoming release.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,17 +1,48 @@
...
+- [pre-commit.ci] pre-commit autoupdate by @pre-commit-ci in https://github.com/Quansight-Labs/accessible-pygments/pull/49
+- Fix contrast failures for high contrast light theme by @gabalafou in https://github.com/Quansight-Labs/accessible-pygments/pull/33
+- Fix broken theme URLs in README, typo in demo html by @meli-lewis in https://github.com/Quansight-Labs/accessible-pygments/pull/53
+- Update code to not get images from placeholder.com by @Carreau in https://github.com/Quansight-Labs/accessible-pygments/pull/51
+- Reflect that rgb colors are in 0-1 and floats. by @Carreau in https://github.com/Quansight-Labs/accessible-pygments/pull/52
+- Small refactor of color utils for readibility and type checkability by @gabalafou in https://github.com/Quansight-Labs/accessible-pygments/pull/55
+- Rename hex to float function by @gabalafou in https://github.com/Quansight-Labs/accessible-pygments/pull/57
+- Set a11y-light default background to #f2f2f2 (light gray) by @gabalafou in https://github.com/Quansight-Labs/accessible-pygments/pull/56-
+
+# New Contributors
```
Comment (gabalafou) [inline CHANGELOG.md:24]: Should this heading be a level 1 or level 2 heading under the version heading?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,17 +1,48 @@
...
+- MAINT - Update pre-commit hooks by @trallard in https://github.com/Quansight-Labs/accessible-pygments/pull/45
+- [pre-commit.ci] pre-commit autoupdate by @pre-commit-ci in https://github.com/Quansight-Labs/accessible-pygments/pull/49
+- Fix contrast failures for high contrast light theme by @gabalafou in https://github.com/Quansight-Labs/accessible-pygments/pull/33
+- Fix broken theme URLs in README, typo in demo html by @meli-lewis in https://github.com/Quansight-Labs/accessible-pygments/pull/53
+- Update code to not get images from placeholder.com by @Carreau in https://github.com/Quansight-Labs/accessible-pygments/pull/51
+- Reflect that rgb colors are in 0-1 and floats. by @Carreau in https://github.com/Quansight-Labs/accessible-pygments/pull/52
+- Small refactor of color utils for readibility and type checkability by @gabalafou in https://github.com/Quansight-Labs/accessible-pygments/pull/55
+- Rename hex to float function by @gabalafou in https://github.com/Quansight-Labs/accessible-pygments/pull/57
+- Set a11y-light default background to #f2f2f2 (light gray) by @gabalafou in https://github.com/Quansight-Labs/accessible-pygments/pull/56-
```
Comment (gabalafou) [inline CHANGELOG.md:22]:
```suggestion
- Set a11y-light default background to #f2f2f2 (light gray) by @gabalafou in https://github.com/Quansight-Labs/accessible-pygments/pull/56
```

**Review by gabalafou — CHANGES_REQUESTED** [review]
Out of curiosity and for future reference, did you update the changelog manually or is there a command that does it?

Also, I didn't realize that the changelog is updated before the version number is bumped. 

I made a few suggested fixes, hence the "request changes"

File: CHANGELOG.md (hunk 1 above)
Comment (trallard) [reply CHANGELOG.md:24]: Level 2 - should have one and only one Level 1.

Good catch

**Comment by trallard** [comment]
I generally use https://github.com/choldgraf/github-activity
or use the "Generate Changelog" feature in GitHub directly when drafting a release. 

In other projects there are other automations/tools we use but here the above work.

**Comment by trallard** [comment]
Ok fixed all the headings @gabalafou

**Review by gabalafou — APPROVED** [review]
Thanks, looks great!

**Pull request merged**